sycl/plugins/native_cpu/CMakeLists.txt

```diff
@@ -23,32 +23,67 @@ add_sycl_plugin(native_cpu
     UnifiedRuntimeCommon
 )
 
+option(OCK_SOURCE_DIR "Root to a local checkout of the oneAPI Construction Kit" "")
+option(OCK_GIT_REPO "Git repository for the oneAPI Construction Kit FetchContent" "")
```

Comment:
I thought option was for a boolean variable https://cmake.org/cmake/help/latest/command/option.html

Reply:
Thanks, I wasn't really aware of that, I changed the non-bool ones to use `set()`.